index.html

```diff
@@ -441,6 +583,17 @@ <h3 id="installation-sec">
         </div>
         <section>
           <h4>
+            <code>onbeforeinstallprompt</code> attribute
```

Comment:
Should this come after onappinstalled, to match the order they appear in the IDL?